staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization_test.go

```diff
@@ -62,19 +62,42 @@ func TestGetAuthorizerAttributes(t *testing.T) {
 			},
 		},
 
-		"resource": {
+		"create resource without namespace": {
 			Verb: "POST",
-			Path: "/api/v1/nodes/mynode",
+			Path: "/api/v1/nodes",
 			ExpectedAttributes: &authorizer.AttributesRecord{
 				Verb:            "create",
-				Path:            "/api/v1/nodes/mynode",
+				Path:            "/api/v1/nodes",
 				ResourceRequest: true,
 				Resource:        "nodes",
 				APIVersion:      "v1",
-				Name:            "mynode",
 			},
 		},
-		"namespaced resource": {
+		"create resource with namespace": {
+			Verb: "POST",
+			Path: "/apis/apps/v1/namespaces/myns/deployments",
+			ExpectedAttributes: &authorizer.AttributesRecord{
+				Verb:            "create",
+				Path:            "/apis/apps/v1/namespaces/myns/deployments",
+				ResourceRequest: true,
+				Namespace:       "myns",
+				Resource:        "deployments",
+				APIVersion:      "v1",
```

Comment:
You are missing `APIGroup:"apps"` in the expected outputted attributes.

Reply:
sorry,i am so careless . APIGroup has been added!